atip/simulator.py

```diff
@@ -1,14 +1,62 @@
 """Module containing an interface with the AT simulator."""
 import logging
+from dataclasses import dataclass
 from warnings import warn
 
 import at
 import numpy
+from numpy.typing import ArrayLike
 import cothread
 from scipy.constants import speed_of_light
 from pytac.exceptions import FieldException
 
 
+@dataclass
+class LatticeData:
+
+    twiss: ArrayLike
+    tunes: ArrayLike
+    chrom: ArrayLike
+    emittance: ArrayLike
+    radint: ArrayLike
+
+
+def calculate_optics(
+    at_lattice: at.Lattice, refpts: ArrayLike, calc_emittance: bool = True
+) -> LatticeData:
+    """Perform physics calculations on lattice.
+
+    It would be possible for this to be faster using linopt2 or linopt4.
```

Comment:
I think the following could be clearer:

> As long as the orbit is calculated using find_orbit6 then the feedback systems appear to work. It would be possible for this method to be faster using linopt2 or linopt4, but the calculated values will be less correct and it will be difficult to assess if this causes a problem.

Perhaps it should also be a comment in the code, rather than part of the docstring.